lib/src/configuration.cc

```diff
@@ -44,19 +44,34 @@ namespace lth_log = leatherman::logging;
     const std::string DEFAULT_SPOOL_DIR { (DATA_DIR / "var" / "spool").string() };
     const std::string PID_DIR { (DATA_DIR / "var" / "run").string() };
     static const std::string DEFAULT_LOG_DIR { (DATA_DIR / "var" / "log").string() };
+
+    static const std::string DEFAULT_MODULES_DIR = []() {
+        wchar_t szPath[MAX_PATH];
+        if (GetModuleFileNameW(NULL, szPath, MAX_PATH) == 0) {
+            throw Configuration::Error { "failed to retrive pxp-agent binary path" };
+        }
+        // Go up twice, assuming the subtree from Puppet Specs:
```

Comment:
Should this also be made to work with the normal make install? I guess we don't want to have a generic modules directory, so I don't see that working.